app/templates/components/search-card.hbs

```diff
@@ -1,13 +0,0 @@
-<div class="layout-column">
-  <form {{action "searchIssues" params on="submit"}}>
```

Comment:
search functionality is being removed.
is that a permanent change?  It cant be done?

Reply:
It can be done. However, for the flexibility of adding a new org, and sync data via wikidata. It is the viable solution for phase 1.

Reply:
Ya, I agree that search should be removed until phase 2.